README.md

```diff
@@ -129,7 +176,7 @@ SQL database. If you need an example, you can use `people` and `wishes` models.
 Your answer...
 
 ```
-Replace this with your answer
+A schema is a framework for how things should be organized. For example a person can have more than one wish(one to many)
```

Comment:
Schema is a definition of database table structure. A one to many relationship is modeled by a reference to the id of the one (`person.id`) on each entity of the many (`wish.person_id`)